sklearn/feature_selection/_sequential.py

```diff
@@ -77,14 +77,17 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin, BaseEstimator
         Whether to perform forward selection or backward selection.
 
     scoring : str or callable, default=None
-        A single str (see :ref:`scoring_parameter`) or a callable
-        (see :ref:`scoring_callable`) to evaluate the predictions on the test set.
+        Scoring method to use for cross-validation. Options:
+
+        - str: see :ref:`scoring_string_names` for options.
+        - callable: a scorer callable object (e.g., function) with signature
+          ``scorer(estimator, X, y)``. See :ref:`scoring_callable` for details.
+        - `None`: the `estimator`'s
+          :ref:`default evaluation criterion <scoring_api_overview>` is used.
 
         NOTE that when using a custom scorer, it should return a single
         value.
```

Comment:
This constraint could be integrated with the description of callable a few lines above: 

"callable with [signature ...], that returns a single value"